docs/best-practices.md

```diff
@@ -238,6 +236,31 @@ useful server-side features that might be useful for most of the use cases.
    .serve(":8080", service)
  ```
 
+### Finagle Filters vs. Finch Endpoints
+
+Finch endpoints are designed to be able to substitute (when it's reasonable) both Finagle
+services and filters. While it's totally clear that an `Endpoint` in Finch is a core
+abstraction and it should be used whenever a `Service` is used in Finagle, it's not so obvious
```

Comment:
"While it's  clear that an `Endpoint` in Finch is a core abstraction and can be thought of as analogous to a Finagle `Service`, it's not always clear when you should use an endpoint over a Finagle `Filter` for things such as authentication."